### DISCUSSION TITLE
Deleting Tag not updating Note

### DISCUSSION THREAD

**Discussion by DavidMFCS** [RafaelGB/obsidian-db-folder#709] [Q&A]
### What happened?

When I click on the 'x' to delete a tag value the tag is not removed from the note. Is there something I am missing or not doing to allow values to be deleted?

### What platform were you using?

Desktop

### Version of the plugin

3.1.2

### Are you using the latest version of the plugin in your Obsidian vault?

- [X] I have verified that I am on the latest version

### Are you check if there is a similar issue?

- [X] I have verified that there are not similar issues

**Comment by RafaelGB** [comment]
You have to close the menu previously (esc) to enable the remove buttons. We are searching a better way to manage the blur event
